pkgs/tools/misc/zellij/default.nix

```diff
@@ -22,13 +22,17 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
-    installShellCompletion assets/completions/zellij.{bash,fish} --zsh assets/completions/_zellij
+    mkdir -p assets/completions
+    for shell in fish bash zsh; do
+      $out/bin/zellij generate-completion $shell > assets/completions/zellij.$shell
+    done
+    installShellCompletion assets/completions/zellij.{bash,fish} --zsh assets/completions/zellij.zsh
   '';
 
   meta = with lib; {
     description = "A terminal workspace with batteries included";
     homepage = "https://zellij.dev/";
     license = with licenses; [ mit ];
-    maintainers = with maintainers; [ therealansh ];
+    maintainers = with maintainers; [ therealansh _0x4A6F ];
```

Comment:
This is a tiny thing, but usually, updating a package and adding a maintainer is done in a separate commit.